tests/Unit/Tasks/TaskObserverDeleteTest.php

```
@@ -129,6 +91,60 @@ public function it_force_delete_removes_relations_from_database()
+
+        /* Assert */
+        $this->assertDatabaseMissing('tasks', [
+            'id' => $taskId,
+        ]);
+    }
+
+    #[Test]
+    public function it_deletes_tasks_soft_deletes()
+    {
+        /* Arrange */
+        $document = $this->task->documents()->first();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$document'. (undefined)

(UnusedLocalVariable)